enhancements/console/dynamic-plugins.md

```diff
@@ -272,6 +272,35 @@ available and the user must refresh their browser to see the changes. (We won't
 refresh the page automatically to avoid possibly losing data if the user is
 entering something into a form or the YAML editor.)
 
+### Localization
+
+Console's frontend will use `/locales/resource.json?lng={{lng}}&ns={{ns}}`
+as i18next loadPath.
```

Comment:
This seems like an implementation detail that we can leave out of this enhancement since it has no impact on the dynamic plugins themselves. It prevents us from having to update the enhancement proposal is the path ever changes.